checkov/main.py

```diff
@@ -467,6 +471,17 @@ def add_parser_args(parser: ArgumentParser) -> None:
                     'custom policies and suppressions if using an API token. Note: it will prevent BC platform IDs from '
                     'being available in Checkov.',
                action='store_true')
+    parser.add('--use-platform-enforcement-rules', action='store_true',
```

Comment:
@mikeurbanski1 I don't like the word `platform` here. Perhaps `use-enforcement-rules` and then in the description explain it is configured in either Bridgecrew or Prisma Cloud? IDK

Reply:
Yeah, I was 50/50 on this. I will remove platform.